### PR TITLE
update combo images

### PR DESCRIPTION
**Pull request by didiyordanova**

### CODE REVIEW

**Comment by didiyordanova** [comment]
not ready for review

**Comment by didiyordanova** [comment]
@yordan-mitev this is ready for a review :)

File: controls/combobox/editmode-and-search.md [hunk 1]
```diff
@@ -28,29 +28,29 @@ ComboBox provides both case-sensitive and case-insensitive searching modes. The
 * `HighlightTextColor`(*Microsoft.Maui.Graphics.Color*)&mdash;Defines the color of the text that will be highlighted when searching is performed.
 
 > Searching can be performed when `IsEditable` is `true`.
+> When binding to a complex objects, ComboBox `DisplayMemberPath` property needs to be set. Also when `IsEditable` is `true` `SearchTextPath` property needs be set.
```
Comment (yordan-mitev) [inline controls/combobox/editmode-and-search.md:31]:
```suggestion
> The ComboBox `DisplayMemberPath` property must be set when binding to complex objects. Also, when `IsEditable` is `true`, the `SearchTextPath` property must be set.
```

File: controls/combobox/editmode-and-search.md [hunk 2]
```diff
@@ -28,29 +28,29 @@ ComboBox provides both case-sensitive and case-insensitive searching modes. The
...
 
 If you want the text to be highlighted when using custom ItemTemplate, you have to add a `RadHighlightLabel` inside the ItemTemplate.
```
Comment (yordan-mitev) [inline controls/combobox/editmode-and-search.md:33]:
```suggestion
If you want the text to be highlighted when using a custom `ItemTemplate`, you have to add a `RadHighlightLabel` inside the `ItemTemplate`.
```

File: controls/combobox/selection.md [hunk 3]
```diff
@@ -27,46 +27,51 @@ The default `SelectinMode`(enumeration of type `Telerik.Maui.Controls.ComboBoxSe
 
 ### Example with Single Selection and SelectedIndex set
```
Comment (yordan-mitev) [inline controls/combobox/selection.md:28]:
```suggestion
### Example with Single Selection and SelectedIndex Set
```

File: controls/combobox/styling.md [hunk 4]
```diff
@@ -10,41 +10,42 @@ slug: combobox-styling
...
+* `PlaceholderColor` (`Microsoft.Maui.Graphics.Color`)&mdash;Defines the color for the placeholder text.
+* `TextColor` (`Microsoft.Maui.Graphics.Color`)&mdash;Defines the color of the text when the control is editable and the color of the selected item when the control is not editable and the selection mode is single.
+* `BackgroundColor` (`Microsoft.Maui.Graphics.Color`)&mdash;Defines the background color of the control.
+* `BorderColor` (`Microsoft.Maui.Graphics.Color`)&mdash;Defines the color of the border.
+* `BorderThickness` (`Microsoft.Maui.Thickness`)&mdash;Defines the thickness of the border.
+* `ClearButtonStyle` (of type `Style` with target type `Telerik.Maui.Controls.RadButton`)&mdash;Defines the style for the clear button.
+* `Font Options` (`FontAttributes`, `FontFamily`, `FontSize`)&mdash;Define the font options to the text of the RadComboBox. It's applied to the Placeholder, Selected Text(for single selection) and when the control is in Editable Mode.
```
Comment (yordan-mitev) [inline controls/combobox/styling.md:19]:
```suggestion
* `Font Options` (`FontAttributes`, `FontFamily`, `FontSize`)&mdash;Define the font options to the text of the RadComboBox. It's applied to the Placeholder, Selected Text (for single selection), and when the control is in Editable Mode.
```

File: controls/combobox/styling.md [hunk 5]
```diff
@@ -10,41 +10,42 @@ slug: combobox-styling
...
+* `HighlightTextColor` (`Color`)&mdash;Specifies the color of the text that will be highlighted when searching isperformed (`IsEditable` has to be `True`).
```
Comment (yordan-mitev) [inline controls/combobox/styling.md:20]:
```suggestion
* `HighlightTextColor` (`Color`)&mdash;Specifies the color of the text that will be highlighted when searching (`IsEditable` must be `True`).
```

File: controls/combobox/templates.md [hunk 6]
```diff
@@ -21,41 +21,41 @@ If the default templates of the control do not suit your needs, you can define c
 
 ### Example with ItemTemplate and SelectedItemTemplate
 
-Here is the [.NET MAUI ComboBox]({%slug combobox-overview%}) definition in XAML:
+**1.** Define the [.NET MAUI ComboBox]({%slug combobox-overview%}) in XAML:
 
 <snippet id='combobox-item-selecteditem-templates'/>
 
-you need to add the `telerik` namespace:
+**2.** Ddd the `telerik` namespace:
```
Comment (yordan-mitev) [inline controls/combobox/templates.md:28]:
```suggestion
**2.** Add the `telerik` namespace:
```

**Review by yordan-mitev — APPROVED** [review]

**Pull request merged**